packages/core/src/transports/offline.ts

```diff
@@ -131,10 +131,10 @@ export function makeOfflineTransport<TO>(
         retryDelay = START_DELAY;
         return result;
       } catch (e) {
-        if (store && (await shouldQueue(envelope, e, retryDelay))) {
+        if (store && (await shouldQueue(envelope, e as Error, retryDelay))) {
```

Comment:
l: Is this a safe type cast? I guess it should be ok but just to double check

Reply:
it's not really safe, it's just that previously `catch e` was typed as `any` I believe, and in more recent versions they have improved on this by making `e: unknown` in a catch block, which is technically more correct as _anything_ can be thrown (think `throw "aaa"`). So it's as safe as it was before 😅